tmt/steps/execute/__init__.py

```diff
@@ -333,3 +358,11 @@ def timeout_hint(self, test):
     def results(self):
         """ Return test results """
         raise NotImplementedError
+
+
+@dataclass
+class Script:
+    """ Represents a script provided by the internal executor """
+    path: str
+    aliases: List[str] = field(default_factory=list)
+    variables: List[str] = field(default_factory=list)
```

Comment:
Could the naming here be a bit different? At first, I wasn't sure what this is for (are these env variables that are passed to the script?). Took me some time to realise that it's the reboot variable, so something like `related_variables` would make it clearer IMHO.

Reply:
ack, yeah, actually the idea was putting possible env vars related to scripts directly defined here, I will try to think about it more, thanks for the comment

Reply:
renamed to `related_variables`